azure-pipelines.yml

```diff
@@ -0,0 +1,26 @@
+# Starter pipeline
```

Comment:
Just in case - i suggest not to add this pipeline to the root of repo.

Reply:
It is test version